crates/resolver/src/name_server/connection_provider.rs

```diff
@@ -458,3 +510,29 @@ impl Future for ConnectionBackgroundInner {
         }
     }
 }
+
+#[cfg(feature = "tokio-runtime")]
+pub mod default_runtime {
```

Comment:
can we call this tokio_runtime? or does that conflict with tokio's modules?

Reply:
No conflict. Done.